nautobot/dcim/migrations/0022_interface_redundancy_group_data_migration.py

```diff
@@ -0,0 +1,23 @@
+from django.db import migrations
+from nautobot.extras.management import clear_status_choices, populate_status_choices
+
+
+def populate_device_redundancy_group_status(apps, schema_editor):
+    """Create/link default Status records for the InterfaceRedundancyGroup content-type."""
+    populate_status_choices(apps, schema_editor, models=["dcim.InterfaceRedundancyGroup"])
+
+
+def clear_device_redundancy_group_status(apps, schema_editor):
+    """De-link/delete all Status records from the InterfaceRedundancyGroup content-type."""
+    clear_status_choices(apps, schema_editor, models=["dcim.InterfaceRedundancyGroup"])
```

Comment:
`s/device/interface/g`

Reply:
Fixed in commit 
https://github.com/grelleum/nautobot/tree/3db49e1559e848ca45fb7750b13b9972f904bb69